docs/finding_legistar_id.md

```diff
@@ -0,0 +1,31 @@
+# Finding Your Municipality’s Legistar ID
+
+Legistar is the most widely used legislation tracking tool in the nation. We are 
+leveraging its wide coverage to help standardize and streamline part of our onboarding 
```

Comment:
`its` -> `it's` if I remember rules correctly.

Reply:
Actually I think this one _is_ correct, i.e. "its"?... (But English is definitely not my first language LOL) Though I did notice some other place where I thought it was wrong. Anyway, will proofread.

Reply:
How about just "Legistar's" because it avoids the confusion hahahaha

Reply:
😆

Reply:
technically "its" is correct because "it's" is short for "it is" which wouldn't make sense in this context but it's (aha) okay if we just use "Legistar's"